src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java

```diff
@@ -705,6 +704,81 @@ public void testReadBasedAnnotations() throws IOException {
         Assert.assertFalse(vc.get().getFilters().contains(GATKVCFConstants.STRAND_ARTIFACT_FILTER_NAME));
     }
 
+
+    public File createSamWithNsandStrandBias(final int numAlts, final int numNs, final int numRefs) throws IOException {
+        final byte altQuality = 50;
+        final byte refQuality = 30;
+        final File samFile = File.createTempFile("duplex", ".bam");
+        final SAMFileHeader samHeader = M2TestingUtils.createSamHeader();
+        final SAMFileGATKReadWriter writer = M2TestingUtils.getBareBonesSamWriter(samFile, samHeader);
+
+        // create 2 Ts with strand bias
+
+        final List<GATKRead> altReads = M2TestingUtils.createReads(numAlts, M2TestingUtils.DEFAULT_ALT_BASES, samHeader, altQuality, "alt");
+        altReads.forEach(read -> {
+            read.setReadGroup(M2TestingUtils.DEFAULT_READ_GROUP_NAME);
+            read.setMappingQuality(60);
+            read.setIsReverseStrand(false);
+            read.setBases(M2TestingUtils.DEFAULT_ALT_BASES);
+            read.setBaseQualities(M2TestingUtils.getUniformBQArray(altQuality, M2TestingUtils.DEFAULT_READ_LENGTH));
+            writer.addRead(read);
+        });
+
+        // create some reads with Ns
+        final byte[] DEFAULT_N_BASES = "CATCACACTNACTAAGCACACAGAGAATAAT".getBytes();
+
+        final List<GATKRead> NReads = M2TestingUtils.createReads(numNs, DEFAULT_N_BASES, samHeader, altQuality, "N");
+        NReads.forEach(writer::addRead);
+
+        // create some ref reads
+        final List<GATKRead> refReads = M2TestingUtils.createReads(numRefs, M2TestingUtils.DEFAULT_REF_BASES, samHeader, refQuality, "ref");
+        refReads.forEach(writer::addRead);
+
+        writer.close();
+        return samFile;
+    }
+
+    @Test
+    public void testLiquidBiopsy() throws Exception {
```

Comment:
This isn't really a test of liquid biopsy, could you rename the test?
Something like, testStrictStrandBiasAndN?